README.md

```diff
@@ -1,2 +1,26 @@
-# How-to-wrap-text-in-DOTNET-MAUI-RadiButton
-This repository contains a sample demonstrating of  wrap text in .NET MAUI RadiButton.
+# How-to-wrap-text-in-.NET-MAUI-RadiButton
+This repository contains a sample demonstrating of wrapping text using LineBreakMode property in .NET MAUI RadiButton.
+
+## LineBreakMode support in .NET MAUI RadiButton (SfRadiButton)
+The LineBreakModeSupport feature enables flexible text truncation and wrapping options within user interface components, enhancing readability and presentation. It facilitates various modes like WordWrap, CharacterWrap, MiddleTruncation, etc., accommodating diverse layout requirements efficiently.
```

Comment:
give space between "LineBreakModeSupport"

Reply:
I have given space between "LineBreakModeSupport" and updated.